test/babel.ts

```diff
@@ -191,6 +191,29 @@ describe("Babel", function () {
     );
   });
 
+  it("ImportAttribute", function () {
+    const code = [
+      `import foo from 'foo';`,
```

Comment:
Can you parse it too? Or do you have to build the assertions programmatically?

Reply:
Just pushed  3db34e8 with new tests.